app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,27 @@
+
+<%= form_for(resource, as: resource_name, url: user_registration_path(resource_name)) do |f| %>
+<div class="d-flex align-items-center justify-content-between p-2" style= "background: #3778c2;">
+<%= link_to(:back) do %>
+<i class="fa fa-arrow-left" style="color: white"></i>
+<% end %>
+<p class="mt-3" style="color: white">REGISTER</p>
+  <div class="actions">
+    <%= f.submit "Next", class:"btn btn-link text-decoration-none", style:"color: white"%>
+  </div>
+</div>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="form-row">
+   <div class="col">
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder:"Full Name", class:"form-control" %>
+  </div>
+  <div class="col">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder:"Email", class:"form-control" %>
```

Comment:
- [ ] [OPTIONAL] It will be nice to have the email confirmation on the signup page, this helps ensure account security.
### Your design
<img width="316" alt="Screenshot 2023-06-10 at 15 40 30" src="https://github.com/sajibarafatsiddiqui/budget_app/assets/58692590/caad39f5-075b-4bf9-9751-8fce09db867f">